content/realtime/channels.textile

```diff
@@ -236,6 +236,149 @@ channel.subscribe { message in
+
+  In addition, rewind will only apply upon *attaching* to a channel, so any subsequent subscriptions post-attach will not fetch old messages.
+
+  *Rewind by number of messages*
+
+  To rewind by a set number of messages, add @?rewind=NUM_MSG@ to your metadata, where @NUM_MSG@ is the number of messages you wish to get at most. For example, to obtain 1 message from history as you subscribe, you would do the following:
+
+  ```[javascript](code-editor:realtime/rewind)
+    var realtime = new Ably.Realtime('{{API_KEY}}');
+    var channel = realtime.channels.get('[?rewind=1]{{RANDOM_CHANNEL_NAME}}');
+    channel.subscribe(function(message) {
+      alert('Received: ' + message.data);
+    });
+  ```
+
+  ```[nodejs](code-editor:realtime/rewind)
+    var Ably = require('ably');
+    var realtime = new Ably.Realtime('{{API_KEY}}');
+    var channel = realtime.channels.get('[?rewind=1]{{RANDOM_CHANNEL_NAME}}');
+    channel.subscribe(function(message) {
+      console.log("Received: "  message.data);
+    });
+  ```
+
+  ```[ruby]
+    realtime = Ably::Realtime.new('{{API_KEY}}')
+    channel = realtime.channels.get('[?rewind=1]{{RANDOM_CHANNEL_NAME}}')
+    channel.subscribe do |message|
+      puts "Received: #{message.data}"
+    end
+  ```
+
+  ```[java]
+    AblyRealtime realtime = new AblyRealtime("{{API_KEY}}");
+    Channel channel = realtime.channels.get("[?rewind=1]{{RANDOM_CHANNEL_NAME}}");
+    channel.subscribe(new MessageListener() {
+      @Override
+      public void onMessage(Message message) {
+        System.out.println("New messages arrived. " + message.name);
+      }
+    });
+  ```
+
+  ```[csharp]
+    AblyRealtime realtime = new AblyRealtime("{{API_KEY}}");
+    IRealtimeChannel channel = realtime.Channels.Get("[?rewind=1]{{RANDOM_CHANNEL_NAME}}");
+    channel.Subscribe(message => {
+      Console.WriteLine($"Message: {message.Name}:{message.Data} received");
+    });
+  ```
+
+  ```[objc]
+  ARTRealtime *realtime = [[ARTRealtime alloc] initWithKey:@"{{API_KEY}}"];
+  ARTRealtimeChannel *channel = [realtime.channels get:@"[?rewind=1]{{RANDOM_CHANNEL_NAME}}"];
+  [channel subscribe:^(ARTMessage *message) {
+      NSLog(@"Received: %@", message.data);
+  }];
+  ```
+
+  ```[swift]
+  let realtime = ARTRealtime(key: "{{API_KEY}}")
+  let channel = realtime.channels.get("[?rewind=1]{{RANDOM_CHANNEL_NAME}}")
+  channel.subscribe { message in
+      print("Received: \(message.data)")
+  }
+  ```
+
+  *Rewind by period of time*
+
+  To rewind by a set period of time, add @?rewind=TIME@ to your metadata, where @TIME@ is how far back in time you wish to rewind. For example, to obtain 10 seconds of messages from history as you subscribe, you would do the following:
```

Comment:
In the initial documentation there is reference to a "time specifier", which is a way of expressing a time interval. We need to use the same language here. Saying "where @TIME@ is how far back in time you wish to rewind" is too imprecise.